app/controllers/moderate_stories_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ModerateStoriesController < ApplicationController
```

Comment:
```suggestion
class Mod::StoriesController < Mod::ModeratorController
```

We recently added an `app/controllers/mod/` directory. Please move it in there and have it inherit from `Mod::ModeratorController` (you can then remove the explicit call to `require_logged_in_moderator`, the parent controller handles it).

Sorry for the chore of renaming the file, test, and route.